NitroxServer/GameLogic/PlayerManager.cs

```diff
@@ -72,6 +72,15 @@ public IEnumerable<Player> GetAllPlayers()
                 return new MultiplayerSessionReservation(correlationId, rejectedState);
             }
 
+            bool hasSeenPlayerBefore = player != null;
+//#if RELEASE
+            if (hasSeenPlayerBefore && player.Token != authenticationContext.Token)
```

Comment:
Let's make this check configurable, defaulting to false to start.

Reply:
I agree, I much prefer that, I just did that for developmental and testing purposes which is why it was commented out in the commit.